src/ai/backend/manager/services/deployment/actions/create_legacy_deployment.py

```diff
@@ -13,7 +13,7 @@
 class CreateLegacyDeploymentAction(DeploymentBaseAction):
     """Action to create a new legacy deployment(Model Service)."""
 
-    creator: DeploymentCreator
+    request: DeploymentCreateRequest
```

Comment:
This name is strange. It looks like a request is going in as an internal action.